specutils/io/registers.py

```diff
@@ -167,3 +169,45 @@ def _load_user_io():
                     import_module(file[:-3])
                 except ModuleNotFoundError:  # noqa
                     pass
+
+
+def identify_spectrum_format(filename, dtype=Spectrum1D):
+    """ Attempt to identify a spectrum file format
+
+    Given a filename, attempts to identify a valid file format
+    from the list of registered specutils loaders.  Essentially a wrapper for
+    `~astropy.io.registry.identify_format` setting **origin** to ``read`` and
+    **data_class_required** to `~specutils.Spectrum1D`.
+
+    Parameters
+    ----------
+    filename : str
+        The absolute filename to the object
```

Comment:
it doesn't *have* to be absolute, right?  i.e. `../some-file.fits` should work even if it's not the best idea to do that?
```suggestion
        A path to a file to be identified
```